tests/kafkatest/tests/client/consumer_test.py

```diff
@@ -101,7 +101,7 @@ def test_broker_rolling_bounce(self, metadata_quorum=quorum.zk, use_new_coordina
         partition = TopicPartition(self.TOPIC, 0)
 
         producer = self.setup_producer(self.TOPIC)
-        consumer = self.setup_consumer(self.TOPIC, group_protocol=group_protocol)
+        consumer = self.setup_consumer(self.TOPIC, group_protocol=group_protocol, prop_file="metadata.recovery.strategy=none")
```

Comment:
Could you please add a comment? [KAFKA-18194](https://issues.apache.org/jira/browse/KAFKA-18194) contains some discussion on this.